zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -73,10 +81,18 @@ public Response bind(@PathParam("noteId") String noteId, String req) throws IOEx
   @GET
   @Path("interpreter/bind/{noteId}")
   public Response bind(@PathParam("noteId") String noteId) {
-    List<InterpreterSettingListForNoteBind> settingList
-      = new LinkedList<InterpreterSettingListForNoteBind>();
+    Object oprincipal = SecurityUtils.getSubject().getPrincipal();
```

Comment:
OK. will do it shortly

Cordialement.

*Hayssam Saleh, Architecture & Stratégie S.I.
*+33 6 85 71 13 95 | LinkedIn Profile
<http://fr.linkedin.com/pub/hayssam-saleh/0/144/bb8> | ebiznext :)
<http://www.ebiznext.com> | Skype ID <http://hayssam.saleh2>


On Tue, Aug 18, 2015 at 8:30 PM, djoelz <notifications@github.com> wrote:

> In
> zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java
> <https://github.com/apache/incubator-zeppelin/pull/53#discussion_r37335212>
> :
>
> > @@ -73,10 +81,18 @@ public Response bind(@PathParam("noteId") String noteId, String req) throws IOEx
> >    @GET
> >    @Path("interpreter/bind/{noteId}")
> >    public Response bind(@PathParam("noteId") String noteId) {
> > -    List<InterpreterSettingListForNoteBind> settingList
> > -      = new LinkedList<InterpreterSettingListForNoteBind>();
> > +    Object oprincipal = SecurityUtils.getSubject().getPrincipal();
>
> This should be its own method as its reused.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/53/files#r37335212>.
>

-- 
*EBIZNEXT <http://www.ebiznext.com>  Java / .NET **/ DevOps / BigData / UX 
/ Agile*
23 rue Vernet 75008 Paris 
<https://maps.google.com/?q=23%20rue%20Vernet%2075008%20Paris>